reparo/syncer/translate.go

```diff
@@ -29,6 +32,10 @@ func pbBinlogToTxn(binlog *pb.Binlog) (txn *loader.Txn, err error) {
 		txn.DDL = new(loader.DDL)
 		// for table DDL, pb.Binlog.DdlQuery will be "use <db>; create..."
 		txn.DDL.SQL = string(binlog.DdlQuery)
+		txn.DDL.Database, txn.DDL.Table, err = parserSchemaTableFromDDL(txn.DDL.SQL)
```

Comment:
I think we should check `txn.DDL.Database` and `txn.DDL.Table` here

Reply:
done